### DISCUSSION TITLE
How to use props correctly with page layout like ui.header() ?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#989] [Q&A] [answered]
### Question

Hi All,

I wanted to ask, what is the right way to use props in page layout elements like ```ui.header()```?

For ex: I read on the Quasar docs that there is a prop called ```height:hint``` which we can use for adjusting the height of the ```QHeader```

So I am assuming that ```ui.header()``` will be similar and I should use the props in the same way. Ex: I tried 
```ui.header(elevated=True).props('height-hint=150')```
But seems like this has no effect, and the height is not changing. 

So, can you please advise me on what is the right way of using the props on the page layout elements?
Thanks a lot for your help with this.

Regards,
Anindya

**Comment by rodja** [comment, marked as answer]
The `height-hint` prop is only for Quasars server side rendering (SSR). As the documentation states:

> When using SSR, you can optionally hint of the height (in pixels) of the QHeader

NiceGUI does not perform SSR yet (see #663). To set the height you can simply use Tailwind or CSS. For example:

```py
with ui.header().classes('h-64'):
    ui.label('NiceGUI')
```

**Reply by Anindya088** [reply]
Ah, so sorry for this, I missed the SSR part on the doc. Thanks for highlighting this. 
I think the Tailwind classes is perfectly fine in this case.

Thanks a lot again.
Anindya
